Created the first database design.

Terunaga opened the pull request:
# WHAT

Created the first database design.
# WHY

To understand the database design required to create Protospace.

Terunaga committed:
Created the first database design.

sinagaki58 commented on README.rdoc:
```diff
@@ -1,28 +1,51 @@
+# Database creation(tables and the respective columns)
+## users
+* name
+* group
```
groupはいらないですね

sinagaki58 commented on README.rdoc:
```diff
@@ -1,28 +1,51 @@
+* title
+* catch_copy
+* concept
+* tag
```
gem使うので、DB上ではtagというカラムは必要ないです

sinagaki58 commented on README.rdoc:
```diff
@@ -1,28 +1,51 @@
+* tag
+* user_id
+
+## images
```
prototypeに使うものなので、thumbnailsのほうがいいとおもいます

sinagaki58 commented on README.rdoc:
```diff
@@ -1,28 +1,51 @@
+
+## images
+* prototype_id
+
```
enumつかってメインのサムネイルとサブのサムネイルを区別しましょう

sinagaki58 commented on README.rdoc:
```diff
@@ -1,28 +1,51 @@
+## comments
+* prototype_id
+* user_id
+* comment
```
textとかのほうがいいですかね

sinagaki58 commented on README.rdoc:
```diff
@@ -1,28 +1,51 @@
+* has many prototypes
+* has many comments
+* has many likes
+* has many tags
```
userとtagは関連づける必要ないです

Terunaga committed:
users table: deleted group column and added member column.

prototypes table: deleted tag column.

thumbnails table: changed the table name from images to thumbnails, added main_thumbnail and sub_thumbnails columns.

comments table: deleted comment column and added text column.

Relations:
user relations: deleted has many tags and has many images.

prototype relations: deleted has many images and added has many thumbnails.

Terunaga commented:
# WHAT

Edited the database designs.

# WHY

Some columns and relations were unnecessary, and some important columns were not written down.
